packs/feats/incredible-familiar-witch.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>Your familiar is imbued with even more magic than other familiars. You can select a base of six familiar or master abilities each day, instead of four.</p>\n<hr />\n<p><strong>Special</strong> Add the bonus familiar abilities you gain for being a witch to this amount.</p>"
+            "value": "<p>Your familiar is imbued with even more magic than other familiars. You can select a base of six familiar or master abilities each day, instead of four.</p><hr /><p><strong>Special</strong> Add the bonus familiar abilities you gain for being a witch to this amount.</p>"
```

Comment:
Should this bee deleted, since it was merged into Incredible Familiar?

Reply:
I was gonna delete it and redirect it after this was merged, but maybe there's no need for the redirect? It's not linked anywhere in the system, at least.